R/pandoc.R

```diff
@@ -77,6 +77,9 @@ pandoc_convert <- function(input,
   # additional command line options
   args <- c(args, options)
 
+  # Use `--quiet` arg to squelch warnings from 2.0+ Pandoc
```

Comment:
Does this eliminate all pandoc output though? (as many users find it quite helpful to see the full pandoc command line printed)

Reply:
We still get all of the usual output:

```


processing file: test.rmd
  |.........                                                        |  14%
  ordinary text without R code

  |...................                                              |  29%
label: setup (with options) 
List of 1
 $ include: logi FALSE

  |............................                                     |  43%
  ordinary text without R code

  |.....................................                            |  57%
label: cars
  |..............................................                   |  71%
  ordinary text without R code

  |........................................................         |  86%
label: pressure (with options) 
List of 1
 $ echo: logi FALSE

  |.................................................................| 100%
  ordinary text without R code


output file: test.knit.md

/usr/local/bin/pandoc +RTS -K512m -RTS test.utf8.md --to html4 --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash+smart --output test.html --email-obfuscation none --self-contained --standalone --section-divs --template /Library/Frameworks/R.framework/Versions/3.4/Resources/library/rmarkdown/rmd/h/default.html --no-highlight --variable highlightjs=1 --variable 'theme:bootstrap' --include-in-header /var/folders/qz/mfwzxz914n16nj4__bnsbccr0000gn/T//Rtmpe7ognl/rmarkdown-str34bc393639ad.html --mathjax --variable 'mathjax-url:https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML' --quiet 

Output created: test.html
```